dotcom-rendering/src/components/Liveness.importable.tsx

```diff
@@ -33,6 +33,16 @@ const toastRoot: Element | null = !isServer
 	? window.document.getElementById('toast-root')
 	: null;
 
+let timer: NodeJS.Timer | number | undefined;
+const updateTimeElement = (time: HTMLTimeElement) => {
+	clearTimeout(timer);
+	const then = new Date(time.dateTime).getTime();
+	const newText = timeAgo(then);
+	if (!isString(newText) || newText === time.innerText) return;
+	time.innerText = newText;
+	timer = setTimeout(() => updateTimeElement(time), 15_000);
+};
```

Comment:
Had to inline this because `Liveness` does lots of imperative manipulation. Changing the props of an already hydrated island was beyond the scope of this PR.